vue-project-stub-no-pwa/vue.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  configureWebpack: {
+    devtool: 'source-map'
+  },
+  transpileDependencies: [
+    'vuetify'
+  ],
+  publicPath: `/${process.env.VUE_APP_PATH}`,
```

Comment:
Do you think it would be better to just specify the VUE_APP_PATH with a leading / (in env files)?

Reply:
Yea, i think thats a safe bet!

Reply:
Future